Source/ButtonMerchant.swift

```diff
@@ -72,7 +72,12 @@ final public class ButtonMerchant: NSObject {
 
      */
     @objc public static func configure(applicationId: String) {
-        core.applicationId = applicationId
+        guard let validAppId = ApplicationId(applicationId) else {
+            let error = ConfigurationError.invalidApplicationId(appicationId: applicationId)
+            print("⚠️ Button :: \(error.localizedDescription)")
```

Comment:
⚠️

Reply:
I mean... you cant see the log at all -- this gives you some chance